docs/presentations/datafusion-meetup-nyc-2024/talk.qmd

```diff
@@ -369,6 +369,24 @@ Nope. You should use Ibis _with_ `X`.
 
 ::: {.panel-tabset}
 
+### IMDB Top 10
+
+```python
```

Comment:
```suggestion
```{python}
```

Not sure if we want to run this in the build, so we know it's always copypastable.

Reply:
I'm leaning towards no.

While I strongly expect the API usage above to remain unchanged, if it _did_ change, I wouldn't want to edit an older presentation to reflect the newer API.

Reply:
Good point.